src/modules/expense/Expense.tsx

```diff
@@ -0,0 +1,43 @@
+import { styled, Theme } from '@mui/material/styles';
+import { InfoBar } from '../../shared-components/InfoBar';
+import { PriceSection } from './PriceSection';
+import { Form } from './Form';
+import { colors } from 'src/styles/colors';
+import { Title3 } from '../../shared-components/Title3';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  minHeight: `calc(100vh - ${theme.spacing(3)})`,
+  backgroundColor: colors.red[100],
+}));
+
+const StyledDivUpperSide = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  backgroundColor: colors.red[100],
+  minHeight: '300px',
+}));
+
+const StyledDivBottomSide = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  backgroundColor: colors.light[100],
+  marginBottom: '0',
+  borderRadius: '32px 32px 0 0',
+}));
+
+export const Expense = () => {
+  return (
+    <StyledDiv>
+      <StyledDivUpperSide>
+        <InfoBar />
```

Comment:
You should not use InfoBar component. The very top part of the page contains information  that any mobile device will have displayed there that's why it is in the design. However, you should never implement it. 

Instead, you need some kind of header component that contains arrow on the left and the Title3. 
I see that you added the arrow directly to the Title3 shared component so it would be good to change it.
Title3 should still be a shared component however it should only display given text, nothing more that that as the purpose is to re-use it across application.

Reply:
I designed the top bar unnecessarily I mean infoBar. Just to make it exactly the same visual. I know it that part depends on the devices that user have. And the title section I fixed it